### ISSUE TITLE
Try new ensemble approaches

### ISSUE THREAD

**Issue by jose-r-morera** [jose-r-morera/LITE#12]
Currently the results (logits) are averaged for the n classifiers

Try other aggregations or even ways to build the ensemble
